README.md

```diff
@@ -204,7 +205,7 @@ mock](https://github.com/testdouble/contributing-tests/wiki/Partial-Mock)), we
 could have called `td.replace(app.signup, 'onCancel')`, instead.
 
 Remember to call `td.reset()` in an after-each hook (preferably globally so one
-doesn't have to remember to do so in each-and-every test) so that testdouble.js
+doesn't have to remember to do so in each and every test) so that testdouble.js
```

Comment:
The hyphens here seem unnecessary to me, since we're not turning the phrase into a thing by itself.